neurons/miner.py

```diff
@@ -36,29 +36,28 @@ def __init__(self, config=None):
         bt.logging.info(
             "\N{grinning face with smiling eyes}", "Successfully Initialized!"
         )
-        self.client = httpx.AsyncClient(
-            base_url=self.config.neuron.model_endpoint,
-            headers={"Authorization": f"Bearer {self.config.neuron.api_key}"},
-        )
+        self.client = None
+        self.base_url = self.config.neuron.model_endpoint
+
+    async def _stream(self, url, json_data):
+        async with self.client.post(url, json=json_data) as response:
+            async for chunk in response.content:
+                yield chunk
 
     async def create_chat_completion(self, request: Request):
         bt.logging.info("\u2713", "Getting Chat Completion request!")
-        req = self.client.build_request(
-            "POST", "/chat/completions", content=await request.body()
-        )
-        r = await self.client.send(req, stream=True)
+        content = await request.body()
+        json_data = json.loads(content.decode('utf-8'))
```

Comment:
arent you taking a time hit on this since you need to decode the json instead of directly passing it along?